src/actions/actionExport.tsx

```diff
@@ -56,23 +57,62 @@ export const actionChangeExportEmbedScene = register({
       commitToHistory: false,
     };
   },
-  PanelComponent: ({ appState, updateData }) => (
-    <label style={{ display: "flex" }}>
-      <input
-        type="checkbox"
-        checked={appState.exportEmbedScene}
-        onChange={(event) => updateData(event.target.checked)}
-      />{" "}
-      {t("labels.exportEmbedScene")}
-      <Tooltip
-        label={t("labels.exportEmbedScene_details")}
-        position="above"
-        long={true}
-      >
-        <div className="TooltipIcon">{questionCircle}</div>
-      </Tooltip>
-    </label>
-  ),
+  PanelComponent: ({ elements, appState, updateData }) => {
+    const [increasedPngSize, setPngIncreasedSize] = useState(0);
+    const [increasedSvgSize, setSvgIncreasedSize] = useState(0);
+
+    if (appState.exportEmbedScene) {
+      import(/* webpackChunkName: "image" */ "../data/image").then(
+        async (_) => {
+          const incPng = await _.getPngMetatdataSize({
+            metadata: serializeAsJSON(elements, appState),
+          });
+          setPngIncreasedSize(incPng);
+
+          const incSvg = await _.getSvgMetatdataSize({
+            text: serializeAsJSON(elements, appState),
+          });
+          setSvgIncreasedSize(incSvg);
+        },
+      );
+    }
+
+    return (
+      <label style={{ display: "flex" }}>
+        <input
+          type="checkbox"
+          checked={appState.exportEmbedScene}
+          onChange={(event) => updateData(event.target.checked)}
+        />{" "}
+        {t("labels.exportEmbedScene")}
+        {appState.exportEmbedScene && (
+          <Tooltip
+            label={`PNG: ${nFormatter(increasedPngSize, 1)}, SVG: ${nFormatter(
```

Comment:
```suggestion
            label={`PNG: +${nFormatter(increasedPngSize, 1)}, SVG: +${nFormatter(
```

Reply:
Maybe adding `+` to indicate that this will be extra..

Reply:
My bad :|
Fixed.